Build fails due to API change

From this PR: https://github.com/edgexfoundry/edgex-go/pull/724 a breaking API change was introduced which causes this project to no longer compile. See the delhi jenkins jobs for example: https://jenkins.edgexfoundry.org/view/security/job/security-api-gateway-delhi-stage-go/1/console

```
02:27:38 + make test
02:27:38 make: `test' is up to date.
02:27:38 [security-api-gateway] $ /bin/sh -xe /tmp/jenkins8684344082220762792.sh
02:27:38 + make build
02:27:38 cd core && CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build  -o  edgexproxy .
02:27:44 # github.com/edgexfoundry/security-api-gateway/core
02:27:44 ./main.go:33:25: not enough arguments in call to logger.NewClient
02:27:44 	have (string, bool, string)
02:27:44 	want (string, bool, string, string)
02:27:44 make: *** [build] Error 2
02:27:44 Build step 'Execute shell' marked build as failure
```
I propose that we use a default logging level of `INFO` (i.e. `logger.InfoLog`) as the third argument here: https://github.com/edgexfoundry/security-api-gateway/blob/master/core/main.go#L33.

--- a/.gitignore
+++ b/.gitignore
@@ -8,3 +8,5 @@ glide.lock
 *.log
 *.exe
 edgexsecurity
+
+core/edgexproxy

--- a/core/main.go
+++ b/core/main.go
@@ -30,7 +30,7 @@ import (
 var lc = CreateLogging()
 
 func CreateLogging() logger.LoggingClient {
-	return logger.NewClient(SecurityService, false, fmt.Sprintf("%s-%s.log", SecurityService, time.Now().Format("2006-01-02")))
+	return logger.NewClient(SecurityService, false, fmt.Sprintf("%s-%s.log", SecurityService, time.Now().Format("2006-01-02")), logger.InfoLog)
 }
 
 func main() {

--- a/glide.yaml
+++ b/glide.yaml
@@ -7,5 +7,6 @@ import:
 - package: github.com/dgrijalva/jwt-go
   version: v3.2.0
 - package: github.com/edgexfoundry/edgex-go
+  version: delhi
   subpackages:
   - pkg/clients/logging

